tests/unit/api/test_credential.py

```diff
@@ -211,6 +211,11 @@ async def test_credential_update(credential, snowflake_feature_store, persistent
                 np.nan,
                 str(snowflake_feature_store.id),
             ),
+            # This should not be here because the model should default to []
+            #   But we have 2 methods of creating a credential,
+            #   The python way does not default the group_ids to []
+            #   The http way does
+            ("INSERT", 'insert: "sf_featurestore"', "group_ids", np.nan, []),
```

Comment:
Since we introduce a new field in the model, i expected this item will be here for newly created records.

Reply:
oh I think i get it, I thought it was the update call at the end. My bad